net.hachune.mikutter.yaml

```diff
@@ -0,0 +1,72 @@
+app-id: net.hachune.mikutter
+runtime: org.gnome.Platform
+runtime-version: 3.28
+sdk: org.gnome.Sdk
+command: mikutter
+finish-args:
+  # X11 + XShm access
+  - --share=ipc
+  - --socket=x11
+  # Wayland access
+  - --socket=wayland
+  - --socket=pulseaudio
+  # Notification access
+  - --talk-name=org.freedesktop.Notifications
+  - --share=network
+  - --persist=.mikutter
+build-options:
+  cflags: -O2
+  cxxflags: -O2
+cleanup:
+  - /include
+  - /lib/pkgconfig
+  - /lib/debug
+  - /share/man
+  - /share/ri
+  - /share/doc
+  - /cache
+  - '*.la'
+  - '*.a'
+modules:
+  - name: libyaml
+    sources:
+      - type: archive
+        url: 'http://pyyaml.org/download/libyaml/yaml-0.1.7.tar.gz'
+        sha256: 8088e457264a98ba451a90b8661fcb4f9d6f478f7265d48322a196cec2480729
+
+  - name: ruby
+    config-opts:
+      - --disable-install-doc
+    sources:
+      - type: archive
+        url: 'https://cache.ruby-lang.org/pub/ruby/2.5/ruby-2.5.1.tar.gz'
+        sha256: dac81822325b79c3ba9532b048c2123357d3310b2b40024202f360251d9829b1
+
+  - name: bundler
+    buildsystem: simple
+    build-commands:
+      - gem install --local bundler-1.16.2.gem
+    sources:
+      - type: file
+        url: 'https://rubygems.org/downloads/bundler-1.16.2.gem'
+        sha256: 3bb53e03db0a8008161eb4c816ccd317120d3c415ba6fee6f90bbc7f7eec8690
+
+  - net.hachune.mikutter.rubygems-module.json
+
+  - name: mikutter
+    buildsystem: simple
+    build-commands:
+      - install -Dm644 -t /app/share/mikutter mikutter.rb LICENSE README
```

Comment:
No need to install LICENSE and README unless they are displayed in-app; all files are accessible as part of `${app_id}.Source` extension.